pkg/skyenv/values_windows.go

```diff
@@ -21,3 +19,13 @@ func PackageConfig() PkgConfig {
 	pkgconfig.Hypervisor.EnableAuth = true
 	return pkgconfig
 }
+
+// UserConfig contains installation paths (for windows)
+func UserConfig() PkgConfig {
+	var usrconfig PkgConfig
+	usrconfig.Launcher.BinPath = "C:/Program Files/Skywire/apps"
+	usrconfig.LocalPath = HomePath() + "/.skywire/local"
+	usrconfig.Hypervisor.DbPath = HomePath() + "/.skywire/users.db" //permissions errors if the process is not run as root.
```

Comment:
Same question about the validity of the comment as above.

Reply:
accidental inclusion, and the original comment issue is now basically resolved